src/service/database/index.ts

```diff
@@ -46,9 +53,48 @@ export class DatabaseService {
 
     for (const row of data) {
       const [name, description] = row as string[];
-      databases.push(new DatabaseModel({ name, description }));
+      databases.push(new DatabaseModel({ name, description }, this.context));
     }
 
     return databases;
   }
+
+  async create(
+    name: string,
+    region: string | undefined = undefined,
```

Comment:
do not need to fallback to `undefined` as it is `undefined` by default

Reply:
to many arguments for one function. It is better to have first name param and second object argument with all other optional parameters.